### PR TITLE
Func2

### PR DESCRIPTION
**Pull request by akashsen-aurx**
you have to approve it if it is right

### CODE REVIEW

File: Cow.txt [hunk 1]
```diff
@@ -3,4 +3,7 @@
 3. she khown as our mother;
 4. she is pure with heart.
 5. here colour may be white,black,red,ornage,etc.
-6. cow is usefull.
+6. Cow is also known as 'Guy'.
```
Comment (ravindra-aurx) [inline Cow.txt:6]: cow is not a mata not a guy?

File: func2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is Func2 file Develope by Akash
```
Comment (ravindra-aurx) [inline func2.txt:1]: isme bhi kuch galti hai

**Review by ravindra-aurx — CHANGES_REQUESTED** [review]

File: Cow.txt (hunk 1 above)
Comment (akashsen-aurx) [reply Cow.txt:6]: okk i will change.

File: func2.txt (hunk 2 above)
Comment (akashsen-aurx) [reply func2.txt:1]: their is no mistake

**Review by ravindra-aurx — APPROVED** [review]

**Pull request merged**